L/libCppInterOp/build_tarballs.jl

```diff
@@ -8,13 +8,13 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "llvm.jl"))
 
 name = "libCppInterOp"
-version = v"0.1.3"
+version = v"0.1.4"
 
-llvm_versions = [v"17.0.6"]
+llvm_versions = [v"18.1.7"]
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/compiler-research/CppInterOp.git", "ecbffafe0016022e40b7dea6e7197ee062dde38f"),
+    GitSource("https://github.com/Gnimuc/CppInterOp.git", "8d79c1a0cafe5d2ff6c74afbcfbd1be320caa06b"),
```

Comment:
It would be a bit longer to get things merged in CppInterOp upstream, so I'd like to temporally switch back to the fork.

Reply:
Yes. I'd like to use that branch as the git source temporarily before it finally gets merged. 

At the moment, I only need one version for testing Julia1.12 with LLVM18.